requirements/common.txt

```diff
@@ -6,3 +6,4 @@ sentencepiece
 pillow
 captum
 packaging
+transformers==2.10.0
```

Comment:
We should keep this in `developers.txt`. `transformers` is not a runtime dependency for TS. It is only required as a test dependency.